devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy

```diff
@@ -204,6 +204,7 @@ def configure() {
 
 	// OnOff minReportTime 0 seconds, maxReportTime 5 min. Reporting interval if no activity
 	refresh()
+	updateColor()
```

Comment:
You need to return a list of all the commands you want to send to the device. As written this will only send the `updateColor` commands and not the `refresh` commands.

Reply:
@tpmanley thank you kindly for the review! I've done what you were asking but the issue still occurred. I wrote more in a comment in Jira ticket.

Reply:
@tpmanley I've added the last changes by adding initial settings in installed for this bulb. Could you please review? 
And thanks for the support! :)